src/pfe/file-watcher/scripts/liberty-container.sh

```diff
@@ -261,10 +261,14 @@ elif [ "$COMMAND" == "update" ]; then
 	# get the status code of the build, if the build failed, update the status
 	if [[ $exitCode -ne 0 ]]; then
 		echo "idc build failed for $ROOT with exit code $exitCode $(date)"
-		if [[ $exitCode -eq 7 ]]; then
+		if [[ $exitCode -eq 1 ]]; then
```

Comment:
Dont think we should handle exit code 1 here, we can handle it in projectUtil.executeBuildScript function. The local if block is missing this error handling but it is present for K8, maybe that explains why it was seen only on local. We can add that if block for ec 1 over there; so projectUtil can handle any issues if script execution goes wrong and exits w/1

Reply:
As long as the return code is non-zero. we need to exit a non-zero exit code. Otherwise, the `processManager.spawnDetached` won't catch the failure. 
The update case should be the same as create case: https://github.com/eclipse/codewind/blob/390e587c030d3c063093109d429a4990eb9da1ff/src/pfe/file-watcher/scripts/liberty-container.sh#L212